main.sh

```diff
@@ -16,21 +16,13 @@ function download {
 function extract {
     [ -e $SCHEDULE ] && hash_old=$(md5sum $SCHEDULE)
     rm -f $SCHEDULE
-    pdftotext -f 1 -layout $TEMP/nea.pdf $TEMP/raw.txt
+    pdftotext -f 1 -l 1 -layout $TEMP/nea.pdf $TEMP/raw.txt
     sed -n '/;d"x÷af/,/;d"x–@/p' $TEMP/raw.txt > $TEMP/part.txt
     sed -i 's/\;d"x–.//; /M/!d; s/^ \+//' $TEMP/part.txt
-    # NOTE: 2utf8 is not-really required but for fail and debug situations
```

Comment:
@munikarmanish thats why its commented, `2utf8` was problematic for normal users, so It was dropped.